src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -54,7 +54,7 @@ public abstract class OptionsBase : TrainerInputBaseWithWeight
                 ShortName = "m, MemorySize", SortOrder = 50)]
             [TGUI(Description = "Memory size for L-BFGS", SuggestedSweeps = "5,20,50")]
             [TlcModule.SweepableDiscreteParamAttribute("MemorySize", new object[] { 5, 20, 50 })]
-            public int IterationsToRemember = Defaults.IterationsToRemember;
+            public int HistorySize = Defaults.HistorySize;
```

Comment:
nit:
```suggestion
            public int HistoryLength = Defaults.HistorySize;
```
`Size` sounds like the number of parameters we can store.